### ISSUE TITLE
Clock disappears upon hiding the Windows desktop

### ISSUE THREAD

**Issue by tdnzr** [danielchalmers/DesktopClock#46]
When I hide the Windows desktop via the `Win + D` shortcut, the DesktopClock is hidden alongside all other windows. I'd hoped the clock would stay visible in precisely this situation.

**Comment by danielchalmers** [comment]
I don't see that. Is `Stay on top` enabled? What version of Windows and DesktopClock?

**Comment by tdnzr** [comment]
No, `Stay on top` is disabled. I wanted the clock to only be visible when I minimize all my app windows via `Win + D` and look at the desktop, not be visible permanently. (To give an example: Rainmeter skins are only visible on the desktop, not drawn on top of other windows; but conversely, these skins aren't hidden alongside all other app windows when I use `Win + D`.)

Re: versions: up-to-date Windows 11 (23H2) and the most recent version of DesktopClock as of when I posted the issue.

**Comment by danielchalmers** [comment]
> I would also like to request this feature.
> 
> For your reference, here is some info on how Rainmeter keeps windows from being minimized when Show Desktop is activated:
> 
> [rainmeter/rainmeter#339 (comment)](https://github.com/rainmeter/rainmeter/issues/339#issuecomment-1477088235)

@realAllonZ Thanks for that, very helpful to see another project making it work! It looks like a better version of what I did in [DesktopWidgets](https://github.com/danielchalmers/DesktopWidgets/blob/e6ba6d42d2de48417592472302daa3975723decb/DesktopWidgets/Classes/ForegroundTracker.cs) with less wibbly-wobbly, timey-wimey stuff.

I'll keep this in mind for when I get around to making a new release and see if anyone has more thoughts in the meantime 🙂

**Comment by sungnix** [comment]
I also would also like to request this feature.
[Stay on top] is disabled, When I hide the Windows desktop via the Win + D shortcut to show desktop, the DesktopClock is hidden alongside all other windows.
